dist/tools/travis-scripts/build_and_test.sh

```diff
@@ -7,6 +7,8 @@
 # directory for more details.
 #
 
+CI_BASE_BRANCH=${CI_BASE_BRANCH:-master}
```

Comment:
It looked to me like all tools take the base branch as parameter.
(Within murdock, that variable is exported into the environment anyways.)